data.md

```diff
@@ -179,11 +190,11 @@ All data is also available on [our GitHub repository for analysis](https://githu
           rows.forEach((row, rowIndex) => {
             const columns = row.split(',');
             tableHTML += '<tr>';
-            columns.forEach(column => {
+            columns.slice(0, 10).forEach(column => {
```

Comment:
This is the only thing I would not do here. I think it's very valuable to be able to see all columns, even if they are a lot, as it can show to someone whether they are interested in this file or not

Reply:
Agreed! I remoted the limit and instead excluded the index columns to save space for many columns.